codespell_lib/data/dictionary.txt

```diff
@@ -77,20 +105,45 @@ abnornally->abnormally
 abnove->above
 abnrormal->abnormal
 aboce->above, abode,
+abodmen->abdomen
+abodminal->abdominal
 aboluste->absolute
 abolustely->absolutely
 abolute->absolute
+abomanation->abomination
+abominacion->abomination
+abominaton->abomination
+abomonation->abomination
 abondon->abandon
 abondoned->abandoned
 abondoning->abandoning
 abondons->abandons
+abonimation->abomination
 aboout->about
 abord->abort, aboard,
+aborginial->aboriginal
+aboriganal->aboriginal
+aborigenal->aboriginal
 aborigene->aborigine
+aboriginial->aboriginal
+aborigional->aboriginal
+aborignial->aboriginal
+aborigonal->aboriginal
+aboroginal->aboriginal
 aborte->aborted, abort, aborts,
 abortificant->abortifacient
 aboslute->absolute
 aboslutely->absolutely
+aboslutes->absolutes
```

Comment:
`absolute` is an adjective, why the plural here?

Of course, adjectives can be used as nouns in spoken English, but that's not really correct English, is it? In any case, the [nominalized](https://www.merriam-webster.com/dictionary/nominalize) or [substantivized](https://www.merriam-webster.com/dictionary/substantivize) adjective usually remains singular. There have already been debates on the subject here. See:
* [Adjectives that Look Like Nouns](https://www.merriam-webster.com/words-at-play/adjectives-that-look-like-nouns)
* [Adjectives Function as Nouns](http://myenglishgrammar.com/lesson-3-adjectives/4-adjectives-function-as-nouns.html)
* [Are adjectives the new nouns?](https://www.macmillandictionaryblog.com/are-adjectives-the-new-nouns)
* [Advanced English Lesson: Using ADJECTIVES as NOUNS](https://www.engvid.com/adjectives-as-nouns/)
* [How to use Adjectives as Nouns](https://worldenglishblog.com/use-adjectives-as-nouns/)

But then I am not a native speaker.